prow/cluster/jobs/istio-releases/pipeline/istio-releases.pipeline.master.yaml

```diff
@@ -102,6 +102,8 @@ presubmits:
         - prow/istio-unit-tests.sh
       nodeSelector:
         testing: test-pool
+    decoration_config:
+      timeout: 14400000000000 # 4 hours
```

Comment:
@cjwagner 
Can we just write `4h` now? I believe so.

Reply:
Thats correct.